lib/web/scripts/desktop/configure-ad.ps1

```diff
@@ -160,15 +161,16 @@ windows_desktop_service:
     addr:     '{2}'
     domain:   '{3}'
     username: '{4}'
-    server_name: '{5}'
+    sid: '{5}'
+    server_name: '{6}'
     insecure_skip_verify: false
     ldap_ca_cert: |
-{6}
+{7}
   discovery:
     base_dn: '*'
   labels:
-    teleport.internal/resource-id: {7}
-'@ -f $TELEPORT_PROVISION_TOKEN, $TELEPORT_PROXY_PUBLIC_ADDR, $LDAP_ADDR, $DOMAIN_NAME, $LDAP_USERNAME, $COMPUTER_NAME, $CA_CERT_YAML, $TELEPORT_INTERNAL_RESOURCE_ID
+    teleport.internal/resource-id: {8}
```

Comment:
Eight positional arguments make it rather hard to read; perhaps it is worth switching to variable interpolation?

Reply:
61faa7d8219c2797a3a86c024fd947c2f6f6b56e

@LKozlowski just double checking that there wasn't some particular reason we weren't using string interpolation already (?)